plugins/wpe-headless/.gitignore

```diff
@@ -1,3 +1,5 @@
 .DS_Store
 Thumbs.db
-/vendor/
\ No newline at end of file
+/vendor/
+
+.env.testing
```

Comment:
Is it worth also adding these?

```
/tests/_output/
/tests/_support/
```

Reply:
`/tests/_output/` has it's own `.gitignore`. `/tests/_support/` needs to be included as it has the generated parent test classes.